### ISSUE TITLE
Add option for `lando info|config|list` etc to be outputted as JSON

### ISSUE THREAD

**Issue by pianomansam** [lando/lando#1455]
TLDR; `lando info` uses single quotes and thus isn't valid JSON, causing JSON parsing utilities like `jq` to error out.

**Tell us about your setup**

*v3.0.0-rc.9 on Mac OSX 10.14.3*

*jq-1.6*

**Tell us about your `.lando.yml`**

This happens will all app stacks.

**Tell us about the command you were running**

```lando info | jq```

**Tell us about the error you got**

```parse error: Invalid numeric literal at line 3, column 12```

**Tell us generally about your bug**

I'm trying to parse out the information provided by `lando info`. The command's output looks like JSON so I tried parsing it with `jq`, a pretty conventional JSON parsing tool. The problem is, though, that because of the single quotes surrounding the values, `jq` chokes. Non-quoted object properties aren't technically valid JSON either, but `jq` [supports them](https://stedolan.github.io/jq/manual/#ObjectConstruction:{}). It [doesn't support](https://github.com/stedolan/jq/issues/107) single quotes, though.

**Tell us more**

The `lando info` command should output valid JSON which can be parsed by other tools.

**Comment by pirog** [comment]
@pianomansam marking this as a feature instead of bug. Definitely keen to add this and see https://github.com/lando/lando/pull/1457. Will comment over there momentarily! 

Thanks a bunch for contributing!

**Comment by pirog** [comment]
Alright this is merged into https://github.com/lando/lando/releases/tag/v3.0.0-rc.13 @pianomansam
